python/tskit/util.py

```diff
@@ -423,16 +432,55 @@ def unicode_table(rows, title=None, header=None, row_separator=True):
         else:
             if i != 0 and not last_skipped and row_separator:
                 out.append(f"╟{'┼'.join('─' * w for w in widths)}╢\n")
+
             out.append(
-                f"║{row[0].ljust(widths[0])}│"
-                f"{'│'.join(cell.rjust(w) for cell, w in zip(row[1:], widths[1:]))}║\n"
+                "║"
+                + "│".join(
+                    f"{r:{a}{w}}" for r, w, a in zip(row, widths, column_alignments)
+                )
+                + "║\n"
             )
             last_skipped = False
 
     out.append(f"╚{'╧'.join('═' * w for w in widths)}╝\n")
     return "".join(out)
 
 
+def html_table(rows, *, header):
+    """
+    Called e.g. by jupyter notebooks to render tables
```

Comment:
This docstring isn't correct, as this method isn't directly called by jupyter.

Reply:
Yes, I think I wrongly moved that from the `_repr_html_` method (comment above). I guess it should be moved back. I don't think I have write access to this PR any more, as Jerome opened a new branch, so maybe you could do it @benjeffery ?

Reply:
Fixed.

Reply:
Thanks a lot @jeromekelleher